app/controllers/metadata_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MetadataController < ApplicationController
+
+  before_action :check_user_authentication if Rails.env === 'production'
+
+  def index
+    gon.data = {
+      authorization: 'Bearer ' + user_token,
+      id: params[:dataset_id]
```

Comment:
Let's call it dataset_id => `dataset_id: params[:dataset_id]` because the id is the metadata id